src/modules/Dashboard/components/TakeNowModal/LabeledDropdown/LabeledDropdown.tsx

```diff
@@ -0,0 +1,95 @@
+import { Autocomplete, Box, TextField } from '@mui/material';
+import React, { useCallback, useRef, useState } from 'react';
```

Comment:
```suggestion
import { useCallback, useRef, useState } from 'react';
```
Please move the import from React to the top.

Reply:
Hey @IhorNesterenko99. I don't understand this suggestion. Why do I need to move it to the top?

Reply:
Hi @sultanofcardio. If you take a look at the files in the project, the first import is always from React, then the others. It's a good idea to implement highlighting and auto-fixing with ESLint and Prettier, as was suggested by @mbanting.

Reply:
Hi @IhorNesterenko99. I understand the convention for having React as the 1st import of a `.tsx` file, however there's no rule in our lint configuration that dictates this specifically, so IDEs would not automatically do reorder this line, nor show an error about it – at least mine doesn't, and I have both ESLint and Prettier enabled. I'm quite certain @sultanofcardio also has ESLint and Prettier enabled in his dev environment as he's the one who was tasked to unify linting rules across the projects a couple months back.

I haven't figured out how to tweak `.eslintrc`'s `import/order` rules to always ensure `react` import is first unless it's in its own group (separated by an extra newline), which seems unnecessary. But if you have any ideas for automating this I'd welcome it!